tests/dummy/app/pods/docs/guides/header/size-constraints/template.md

```diff
@@ -1,20 +1,20 @@
 # Size Constraints
 
 You can set the `widthConstraint` property on your table to ensure that it never
-grows too big or too small. There are 4 possible settings:
+grows too big or too small. There are six possible settings:
 
 1. `eq-container`: Ensures that the table is always exactly the width the of its
 container.
 
-2. `eq-container-slack`: Similar to `eq-container`, but allocates excess whitespace to an empty slack column on the right side of the table.
+2. `eq-container-slack`: Similar to `eq-container`, but allocates excess whitespace to an empty "slack" column on the right side of the table.
 
 3. `gte-container`: Ensures that the table is always the same width or larger than its container.
 
-4. `eq-container-slack`: Similar to `gte-container`, but allocates excess whitespace to an empty slack column on the right side of the table.
+4. `eq-container-slack`: Similar to `gte-container`, but allocates excess whitespace to an empty "slack" column on the right side of the table.
```

Comment:
I assume this isn't named the exact same as option 2?

```suggestion
4. `gte-container-slack`: Similar to `gte-container`, but allocates excess whitespace to an empty "slack" column on the right side of the table.
```

Reply:
Good catch.